test/lib/ufe/testPixelMoveCmd.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2022 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import mayaUtils
+import ufeUtils
+import fixturesUtils
+from testUtils import assertVectorAlmostEqual, assertVectorNotAlmostEqual
+
+import mayaUsd.lib
+
+from maya import cmds
+from maya import standalone
+
+import ufe
+
+@unittest.skipUnless(mayaUtils.ufeSupportFixLevel() > 0, "Requires pixelMove UFE support.")
+class PixelMoveCmdTestCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        fixturesUtils.readOnlySetUpClass(__file__, initializeStandalone=False)
+
+    def setUp(self):
+        cmds.file(new=True, force=True)
+        cmds.select(clear=True)
+    
+    def testPixelMove(self):
+        '''Pixel move command must move non-Maya UFE objects.'''
+
+        # pixelMove has different behavior for orthographic and perspective
+        # views.  Only perspective views are affected by the selected objects'
```

Comment:
Should we also test doing pixel move while looking through an ortho camera?

Reply:
Ortho camera pixel move is unaffected by the selection, and therefore did not require UFE-related changes.  Testing pixel move with ortho camera is essentially testing Maya behavior, which plugins should simply rely on and expect (hope?) is tested in Maya.